pkg/forwarder/transaction.go

```diff
@@ -302,3 +303,11 @@ func (t *HTTPTransaction) internalProcess(ctx context.Context, client *http.Clie
 	log.Tracef("Successfully posted payload to %q: %s", logURL, string(body))
 	return resp.StatusCode, body, nil
 }
+
+// GetPayloadSize returns the size of the payload.
+func (t *HTTPTransaction) GetPayloadSize() int {
+	if t != nil && t.Payload != nil {
```

Comment:
What is the point of checking `t != nil`? Afaik this will throw a `nil` pointer exception before getting in here if `t == nil`.

Reply:
True, I don't think there's any point to check for `nil`. None of the other methods do either.